### PR TITLE
Fix uv_cpu_info() error on FreeBSD arm platform.

### PR DESCRIPTION
**Pull request by elephantp**
ref #1694

### CODE REVIEW

File: src/unix/freebsd.c [hunk 1]
```diff
@@ -294,8 +295,21 @@ int uv_cpu_info(uv_cpu_info_t** cpu_infos, int* count) {
...
+  /* The key hw.model and hw.clockrate are not available on FreeBSD ARM. */
+  model_key = "hw.machine";
+  cpuspeed = 0;
+#else
+  model_key = "hw.model";
+
+  size = sizeof(cpuspeed);
+  if (sysctlbyname("hw.clockrate", &cpuspeed, &size, NULL, 0)) {
+    return -errno;
+  }
```
Comment (santigimeno) [inline src/unix/freebsd.c]: style: remove the curly braces

Comment (elephantp) [reply src/unix/freebsd.c]: OK, thanks for correction.

**Comment by cjihrig** [comment]
Fresh CI: https://ci.nodejs.org/view/libuv/job/libuv-test-commit/666/

**Comment by cjihrig** [comment]
Landed in 9241cc297767eb36820133532eb4d9d8fa7bcda2. Thanks!

**Pull request closed without merging**